src/foam/u2/view/MultiChoiceView.js

```diff
@@ -167,20 +167,45 @@ foam.CLASS({
       return this.dao.where(this.IN(of.ID, this.data));
     },
 
+    function isChoiceSelected(data, choice){
+      var isSelected = false
+
+      for ( var i = 0 ; i < data.length ; i++ ) {
+        isSelected = foam.util.equals(data[i], choice);
+
+        if ( isSelected ) break;
+      }
+      
+      return isSelected;
+    },
+
+    function getIndexOfChoice(data, choice){
+      var indexDataToAdd = -1;
+                          
+      for ( var i = 0 ; i < data.length ; i++ ) {
+        if ( foam.util.equals(data[i], choice) ){
+          indexDataToAdd  = i;
```

Comment:
Again, why not just get rid of the variable and return i here directly?

Reply:
Yeah my bad the code here previously wasn't in a method and was in-line so the direct returns make sense